src/main/java/net/notfab/lindsey/core/commands/economy/Blackjack.java

```diff
@@ -0,0 +1,89 @@
+package net.notfab.lindsey.core.commands.economy;
+
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.jodah.expiringmap.ExpiringMap;
+import net.notfab.lindsey.core.framework.Emotes;
+import net.notfab.lindsey.core.framework.command.Bundle;
+import net.notfab.lindsey.core.framework.command.Command;
+import net.notfab.lindsey.core.framework.command.CommandDescriptor;
+import net.notfab.lindsey.core.framework.command.Modules;
+import net.notfab.lindsey.core.framework.command.help.HelpArticle;
+import net.notfab.lindsey.core.framework.command.help.HelpPage;
+import net.notfab.lindsey.core.framework.economy.EconomyService;
+import net.notfab.lindsey.core.framework.i18n.Messenger;
+import net.notfab.lindsey.core.framework.i18n.Translator;
+import net.notfab.lindsey.core.framework.models.BlackjackModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Blackjack implements Command {
+
+    @Autowired
+    private Translator i18n;
+
+    @Autowired
+    private Messenger msg;
+
+    @Autowired
+    private EconomyService economy;
+
+    @Autowired
+    private ExpiringMap<Long, BlackjackModel> cache;
+
+    public static int price = 0;
```

Comment:
This variable can cause concurrency issues when multiple users attempt to use the command.